tests/react-a11y-mouse-event-has-key-event/test.tsx.lint

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
```

Comment:
Missing test cases, none of which should have errors _(right?)_:

* `<div />`
* `<div {...props} />`
* `<div onMouseOver={() => {}} {...props} />`
* `<div onMouseOut={() => {}} {...props} />`

Reply:
So the last two do complain, but we'd have to do the type checking in order to extract out those values and check the spread props. Right?

Reply:
Sure, if you want! Two options here:

* Don't do that, and file a followup issue tracking making this an (optionally) type checked rule
* Do that as part of this PR

If&whenever that goes in, the rule should be an `OptionallyTypedRule`, like `unneccessary-bind` is from TSLint core. That way folks can still use the rule if running without a `tsconfig.json` project. 

Does that answer your question?

Reply:
> followup issue tracking making this an (optionally) type checked rule

that works